src/main/java/UC/WeatherRunner/service/RunServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package UC.WeatherRunner.service;
+
+import java.util.List;
+
+import UC.WeatherRunner.entity.RunEntity;
+
+public class RunServiceImpl implements RunService {
```

Comment:
this is the service implementation, the functions should be edited to link up with the database